flir_ptu_viz/CMakeLists.txt

```diff
@@ -4,10 +4,5 @@ project(flir_ptu_viz)
 find_package(catkin REQUIRED COMPONENTS roslaunch)
 catkin_package()
 
-roslaunch_add_file_check(launch/view_model.launch
```

Comment:
Why was this removed?

Reply:
Re-added. The tests were failing previously and I'd just deleted them while I worked on other stuff and forgot to re-add them.